### PR TITLE
Add support for letterSpacing to TextStyle

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

File: packages/flutter/lib/src/painting/text_style.dart [hunk 1]
```diff
@@ -137,7 +144,8 @@ class TextStyle {
       fontWeight: fontWeight,
       fontStyle: fontStyle,
       fontFamily: fontFamily,
-      fontSize: fontSize
+      fontSize: fontSize,
+      letterSpacing: letterSpacing
```
Comment (Hixie) [inline packages/flutter/lib/src/painting/text_style.dart]: you have it after fontStyle everywhere but here

**Comment by Hixie** [comment]
LGTM assuming travis red is because of missing engine bits

File: packages/flutter/lib/src/painting/text_style.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/painting/text_style.dart]: Yeah, the order of the fields is different in TextStyle and ui.TextStyle.  I've matched the callee order in each case.

**Comment by abarth** [comment]
> assuming travis red is because of missing engine bits

Yeah, I'll re-run Travis once I've updated the engine.

**Pull request merged**